src/app/content/hooks/locationChange/trackPageView.ts

```diff
@@ -0,0 +1,11 @@
+import googleAnalyticsClient from '../../../../gateways/googleAnalyticsClient';
```

Comment:
we don't want this `trackPageView` file to be content specific, i'd move it to `src/app/hooks/trackPageView.ts`